GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -3246,13 +3253,22 @@ private void HandleDrop(DragEventArgs e)
                 return;
             }
 
-            foreach (string path in paths)
+            foreach (string path in paths.Where(IsFileExistingInRepo))
             {
+                if (CommitInfoTabControl.SelectedTab != TreeTabPage)
```

Comment:
Why check before modification ? According to [TabControl.cs](https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/TabControl.cs,737), the .net framework already do it.

Reply:
No it does check here: https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/TabControl.cs,795, nor here: https://referencesource.microsoft.com/#System.Windows.Forms/winforms/Managed/System/WinForms/TabControl.cs,811. So this checks avoid executing unnecessary operations.

Reply:
@bansan85 I had a quick look at the decompiled code and I saw that there are some other things done before checking if the tab is already the one selected, as @RussKie pointed out, so I decided to do the test to avoid unnecessary operations.